trawler/src/main/java/io/m9sweeper/trawler/framework/scans/Scanner.java

```diff
@@ -74,4 +90,93 @@ default void run() throws RuntimeException {
             throw new RuntimeException(ex);
         }
     }
-}
\ No newline at end of file
+
+    default String getRegistryAuthorization(DockerRegistry registry) throws Exception {
```

Comment:
I'm fairly certain this all is trivy-specific env vars, right?

Reply:
I there's anything that's not trivy-specific you could extract that part, but most of this is specific just to trivy

Reply:
Most of it isn't trivy-specific: it's just the formatting that is.

this part: `String authorization = "export TRIVY_USERNAME=" + escapeXsi("AWS") + "; ";`